src/MoBi.Presentation/Presenter/EditParametersInContainerPresenter.cs

```diff
@@ -263,10 +270,14 @@ public void Rename(ParameterDTO parameterDTO)
       public void Select(ParameterDTO selectedParameter)
       {
          if (selectedParameter.IsIndividualPreview)
-            return;
+            setupEditPresenter(individualParameterFrom(selectedParameter));
+         else
+            setupEditPresenter(ParameterFrom(selectedParameter));
+      }
 
-         var parameter = ParameterFrom(selectedParameter);
-         setupEditPresenter(parameter);
+      private IndividualParameter individualParameterFrom(ParameterDTO selectedParameter)
+      {
+         return _individualParameterCache.Contains(selectedParameter) ? _individualParameterCache[selectedParameter] : null;
```

Comment:
I think the cache can be initialize to return null if not found so that you don't have this method that does what the casche actually helps with